packages/core/theme-module/theme/pages/Category.vue

```diff
@@ -607,6 +647,20 @@ export default {
         margin-top: var(--spacer-extra-big);
       }
     }
+    &__pagination__options {
```

Comment:
not sure if its safe to have it just like that, sfui examples will be updated soon with new versions